os/arch/arm/src/amebad/Kconfig

```diff
@@ -61,6 +61,30 @@ config AMEBAD_WIFI
 
 endmenu
 
+menu "Realtek RTL8721D BLE Support"
+
+config AMEBAD_BLE
+	bool "Enable Amebad BLE"
+	default n
+	select FTL_ENABLED
+
+config FTL_ENABLED
```

Comment:
`FTL_ENABLED` is selected by `AMEBAD_BLE`, but it has `depends on AMEBAD_BLE`.
It looks dangling. I think we can remove the `depends on`.

Reply:
I have removed the `depends on`.

Reply:
I think it's better to keep `depends on`, because `FTL_ENABLED` only can be opened when `AMEBAD_BLE` on.
If removing `depends on`,  `FTL_ENABLED` can be opened without `AMEBAD_BLE` on.

Reply:
Then, let do like below.
```kconfig
config AMEBAD_BLE
	bool "Enable Amebad BLE"
	default n
	select FTL_ENABLED

if AMEBAD_BLE
config FTL_ENABLED
	bool "Enable FTL"
	default n

config AMEBAD_BLE_PERIPHERAL
	bool "Enable Amebad BLE Peripheral"
	default n

config AMEBAD_BLE_CENTRAL
	bool "Enable Amebad BLE Central"
	default n
endif
endmenu

Reply:
Could you check my previous comment for this?